solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/impl/TestLambdaMARTModel.java

```diff
@@ -127,60 +127,121 @@ public void lambdaMartTestExplain() throws Exception {
     System.out.println(restTestHarness.query("/query" + query.toQueryString()));
   }
 
-  @Test(expected = ModelException.class)
+  @Test
   public void lambdaMartTestNoParams() throws Exception {
-    createModelFromFiles("lambdamart_model_no_params.json",
-        "lambdamart_features.json");
+    final ModelException expectedException = 
+        new ModelException("Model type does not exist org.apache.solr.ltr.ranking.LambdaMARTModel");
```

Comment:
Underneath this seems like a terrible error message to have if the model doesn't have any params. The message should ideally say something like "Model .... could not be built, no params supplied"